searx/settings.yml

```diff
@@ -1377,6 +1377,27 @@ engines:
     about:
       website: https://wiby.me/
 
+  - name: marginalia
+    engine: json_engine
+    shortcut: mar
+    categories: general
+    paging: false
+    # index: {"0": "popular", "1": "blogs", "2": "big_sites",
+    # "3": "default", "4": experimental"}
+    search_url: https://api.marginalia.nu/public/search/{query}?index=4&count=20
```

Comment:
Why is `index=4` the default if it is classified as experimental?

Reply:
Because it is more up to date and does not exclude other sites, which is the case with "big sites" or "blogs".

But anyone can change this to what their preference.

Reply:
Ok thanks for the explanation!

Reply:
FYI: I slightly modified the commit message to include the info from https://github.com/searxng/searxng/pull/1627#discussion_r939704880